unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/UserContextMenu/Scripts/UserContextMenu.cs

```diff
@@ -227,42 +222,43 @@ private void OnAddFriendButtonPressed()
 
     private void OnCancelFriendRequestButtonPressed()
     {
-        OnCancelFriend?.Invoke(userId);
-        CancelFriendRequestAsync(userId).Forget();
+        friendOperationsCancellationTokenSource.Cancel();
```

Comment:
The same concerns: no `Dispose`, repetitions

Reply:
i've added the calls for `Dispose` and `Cancel`